webapp/portlet/src/main/webapp/vue-apps/common/js/FileAttachmentService.js

```diff
@@ -46,3 +46,15 @@ export function getAttachments(objectType, objectId) {
     }
   });
 }
+
+export function getAttachment(objectType, objectId, fileId) {
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/attachments/${objectType}/${objectId}/${fileId}?size=0x0&download=true`, {
+    method: 'GET',
+  }).then(resp => {
+    if (!resp?.ok) {
+      throw new Error(`Error retrieving attachments from server, response code = ${resp.status}`);
+    }
+    return resp;
+  });
+}
+
```

Comment:
```suggestion
```
useless

Reply:
I usedr it here 
https://github.com/Meeds-io/social/blob/8ca095648dc98b1ae8844aec908bd9b6cf9af68d/webapp/portlet/src/main/webapp/vue-apps/attach-image/components/PreviewAttachment.vue#L131
to get the filename from the Http header and the image blob for donwload

Reply:
now, that we have the activity processor to get the file name without REST call, can you please delete this method.